app/src/androidTest/java/com/example/henzoshimada/feeltrip/FeelTripTests.java

```diff
@@ -81,10 +80,48 @@ public void testEditTask() throws DescriptionTooLongException {
             e.printStackTrace();
         }
 
+        Date date = new Date();
         Mood mood = moods.get(0);
         mood.setDescription("test edit");
+        mood.setEmotionalState("happy");
+        mood.setMapPosition(12320.23143, 901273.000);
+        mood.setPrivate();
+        mood.setDate(date);
+        mood.setImage("Mona Lisa");
+        mood.setSocialSit("With friends");
+        mood.setDel();
         editMoodTask.execute(mood);
+    }
 
+    public void testSearchMoodTask(){
+        ArrayList<Mood> moods = new ArrayList<>();
+        ElasticSearchController.GetMoodTask get = new ElasticSearchController.GetMoodTask("description");
+        get.execute("test edit");
+        try {
+            moods.addAll(get.get());
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        } catch (ExecutionException e) {
+            e.printStackTrace();
+        }
+        assertEquals("test getMood", "test edit", moods.get(0).getDescription());
+    }
+
+    public void testDeleteMoodTask(){
```

Comment:
The client might want to delete a mood event they posted, this is just an elegant way of doing so with the Delete.Builder function. We don't necessarily have to implement this